core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -81,7 +81,7 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
             }
             updateAction.map(RequestComplete(StatusCodes.OK, _))
           }
-        }
+        }, Set(AttributeTempTableType.Entity))
```

Comment:
looking for opinions ... should we stick with the `Entity`, `Workspace` after the database code, or go for a syntax something like:
```
dataSource.inTransactionWithAttrTempTable(Entity, Workspace) { dataAccess =>
  ...
}
```
wondering if either is easier to read/write/understand. In the few places we override `TransactionIsolation`, that comes after the database code.

Reply:
I would vote for the latter. I think if I was scanning through code, having them together indicates they're together. I've sometimes thought we should change TransactionIsolation to come before the block, too, but that's out of scope.

Reply:
I don't have an opinion, and David it sounds like you like the change enough to throw it out for feedback--seems like 1.5 votes in favor of the change with none against.